Sources/ParseSwift/Parse Types/File.swift

```diff
@@ -1,12 +1,31 @@
 import Foundation
 
+/**
+  A `File` object representes a file of binary data stored on the Parse server.
+  This can be a image, video, or anything else that an application needs to reference in a non-relational way.
+ */
 public struct File: Saveable, Fetchable {
 
     private let __type: String = "File" // swiftlint:disable:this identifier_name
+
+    /**
+      The name of the file.
+      Before the file is saved, this is the filename given by the user.
+      After the file is saved, that name gets prefixed with a unique identifier.
+     */
+    public var name: String?
+
+    /**
+     The contents of the file.
+     */
     public var data: Data?
+
+    /**
+     The url of the file.
+     */
     public var url: URL?
 
-    public init(data: Data?, url: URL?) {
+    internal init(data: Data?, url: URL?) {
```

Comment:
Should this be internal?

Reply:
There are 2 other initializers that get used publicly, one takes in data, the other takes in url (I'm adding them to the File PR later). The init for data and url are used internally in the objective-c SDK